engine/language-server/src/test/scala/org/enso/languageserver/search/SuggestionsHandlerSpec.scala

```diff
@@ -43,6 +43,7 @@ class SuggestionsHandlerSpec
   import system.dispatcher
 
   val Timeout: FiniteDuration = 10.seconds
+  LoggerSetup.get().setup()
```

Comment:
Why isn't the default `setup` picked up automatically when the SLF4J initializes? Can't we just tell SLF4J to initialize some class that would call `LoggerSetup.get().setup()`? Why don't we have [our own provider](https://www.slf4j.org/manual.html#swapping) to make sure a call to `LoggerSetup.get().setup()` is made?